rubocop-airbnb/config/rubocop-airbnb.yml

```diff
@@ -2,11 +2,6 @@
 # They are custom built for use inside Airbnb and address issues that we have experienced in
 # testing and production.
 
-Airbnb/ClassName:
```

Comment:
We can probably delete class_name.rb and class_name_spec.rb now that we're using Rubocop's `ReflectionClassName` cop instead.